### PR TITLE
Restoration of passages cut out in abridged version

### PR DESCRIPTION
**Pull request by drgrigg**
This a heck of a big pull request! Some 30,000 + words were added, and a lot of chapters renumbered. I see there are some conflicts, but I'll figure out how to resolve those.

### CODE REVIEW

**Comment by drgrigg** [comment]
Alex: I think I've resolved the conflicts so it's ready to merge if you approve.

**Comment by acabal** [comment]
It still says there are conflicts in the PR. Can you take a look?

**Comment by drgrigg** [comment]
Maybe I’m not doing it right. I edited the conflicted files using Github’s online editor and saved them. Anyway, I’ll have another look. There were only three files flagged as having conflicts.

**Comment by drgrigg** [comment]
Alex: when I log on it’s telling me “This branch has no conflicts with the base branch”.

**Comment by acabal** [comment]
That's not what it says for me, it says this branch cannot be rebased on due to conflicts. Try rebasing on to master from your local repo, then fixing any merge commits there, and then force pushing to the PR.

**Comment by drgrigg** [comment]
Rebasing terrifies me, but I think I've done it all correctly. Still seeing the message "This branch has no conflicts with the base branch"

File: src/epub/text/endnotes.xhtml [hunk 1]
```diff
@@ -10,11 +10,14 @@
 			<h2 epub:type="title">Endnotes</h2>
 			<ol>
 				<li id="note-1" epub:type="endnote">
-					<p>Gin: a black woman. <a href="chapter-6.xhtml#noteref-1" epub:type="backlink">↩</a></p>
+					<p>This is the author’s preface to the abridged one-volume edition published in 1889. <cite>Editor</cite>. <a href="preface.xhtml#noteref-1" epub:type="backlink">↩</a></p>
```
Comment (acabal) [inline src/epub/text/endnotes.xhtml:13]: Is "Editor" you? If so, let's use `S.E. Editor` to clarify, with an em dash.

**Review by acabal — CHANGES_REQUESTED** [review]

**Comment by drgrigg** [comment]
OK, fixed the endnote and added an se:role of ann to myself.

**Comment by acabal** [comment]
Excellent, thanks!

**Pull request merged**